src/foam/core/AbstractObjectPropertyInfo.java

```diff
@@ -6,11 +6,25 @@
 
 package foam.core;
 
+import javax.xml.stream.XMLStreamReader;
+import javax.xml.stream.XMLStreamWriter;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
 public abstract class AbstractObjectPropertyInfo
```

Comment:
This was a class that was existing before I worked on the XMLDAO so essentially I just added the methods it would override as part of a generic abstract object class